main/webapp/modules/core/scripts/project/browsing-engine.js

```diff
@@ -234,6 +242,22 @@ BrowsingEngine.prototype.removeFacet = function(facet) {
   }
 };
 
+BrowsingEngine.prototype.hideShowBrowsingFacetSplash = function(clicked) {
```

Comment:
```suggestion
BrowsingEngine.prototype.hideShowFacetSplash = function(clicked) {
```
No need of Browsing is this function name. It's important for the prefs (category of pref) but not here where it's in the parent object (BrowsingEngine). Every thing here is abour browsing.